core/src/robot/Robot.cc

```diff
@@ -705,6 +851,11 @@ namespace jiminy
         return sensorsGroupHolder_;
     }
 
+    Robot::tranmissionssGroupHolder_t const & Robot::getTransmissions(void) const
```

Comment:
`tranmissionss` spelling mistake.

Reply:
Just remove this methode, since `tranmissionssGroupHolder_t` does not exist anyway, and the method is already implemented right above.